en/docs/extended_topics/model_mixed_parallel.md

```diff
@@ -1,12 +1,12 @@
-# Characteristics of parallelism in OneFlow
+# Features of parallelism in OneFlow
 
-In [Consistent and Mirrored view](consistent_mirrored.md), we have already known OneFlow provides mirrored and consistent two point of view and learned about the  `consistent` in OneFlow have some special characteristics.
+In [Consistent and Mirrored view](consistent_mirrored.md), we have already known OneFlow provides mirrored and consistent two point of view and learned about the  `consistent` view in OneFlow have some special features.
 
-Because in `consistent_view`, OneFlow gives the  unified view on logical side. When doing the distributed training, user can choose use data parallelism, model parallelism or hybrid parallelism.
+Because in `consistent_view`, OneFlow gives the  unified view on logical side. When doing the distributed training, user can choose to use data parallelism, model parallelism or hybrid parallelism.
```

Comment:
Because in `consistent_view`, OneFlow provides a logically unified view. During distributed training, users can freely choose ...